doc/debian/jitsi-meet-prosody/prosody.cfg.lua-jvb.example

```diff
@@ -1,3 +1,6 @@
+-- We need this for prosody 13.0
+component_admins_as_room_owners = true
```

Comment:
After seeing this, I wonder if some `jitsi_meet_prosody_config_version = X` could help us migrate configs going forward... just an idea, no need to jump on it.

Reply:
This is prosody config, not ours.

Reply:
Well, I envision rewriting everything and making the upgrade incompatible :) and do a new install with everything being different :) 
I want to get rid of the tenants and the rewriting of every stanza converting room@conference.tenant.domain.com to [tenant]room@conference.domain.com. That is a performance hit and in logic, we keep messing stuff making the real prosody jids leak in the client and break things. 
I want to restructure modules and have a small configuration and one that we can be update ... stick with the docker way of having the virtual host be meet.jitsi. Many good things we have seen through the years. 
But that is me dreaming 🫠